src/IPFSNode.ts

```diff
@@ -32,8 +32,7 @@ const configFile =
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const config = require(configFile);
 
-const log = debug('pinner:ipfs');
-const logError = debug('pinner:ipfs:error');
+const log = debug('pinion:ipfs');
```

Comment:
I'm glad you changed the namespace string.

It was getting confusing, the repo being `pinion`, but the namespace being `pinner` :)